scipy/optimize/__init__.py

```diff
@@ -255,12 +255,21 @@
 
    linprog_verbose_callback -- Sample callback function for linprog (simplex).
 
-Assignment problems:
+Assignment problems
+===================
 
 .. autosummary::
    :toctree: generated/
 
    linear_sum_assignment -- Solves the linear-sum assignment problem.
+   quadratic_assignment -- Approximates solution to the quadratic assignment problem.
```

Comment:
We need to shave 6 characters off this line in order to get green CI. I can't come up with anything better than
`Solves the quadratic assignment problem.`

Any suggestions under 80 chars are welcome.

Reply:
```suggestion
   quadratic_assignment -- Solves the quadratic assignment problem.
```